Doc/library/typing.rst

```diff
@@ -39,12 +39,18 @@ provides backports of these new features to older versions of Python.
 For a summary of deprecated features and a deprecation timeline, please see
 `Deprecation Timeline of Major Features`_.
 
+.. seealso::
+
+   The "Type System Reference" section of https://mypy.readthedocs.io/ -- since
+   the Python typing system is standardised via PEPs, this reference should
+   broadly apply to most Python type checkers, although some parts may still be
+   specific to mypy.
+
 .. seealso::
```

Comment:
Nit: I'd prefer it if we had one `.. seealso` box with multiple links in it, similar to the one we have here, rather than two consecutive `.. seealso` boxes: https://docs.python.org/3/library/stdtypes.html#genericalias.__unpacked__

Reply:
Hmm one thing that makes that box work well is that the entries have a nice bolded title. I could make this just be the URL? Suggestions?

Reply:
I removed the other seealso https://deploy-preview-103100--python-cpython-preview.netlify.app/library/typing.html
I think it's mostly fine, but it's not as nice as the seealso you link because it doesn't have bold headings. I don't really have good heading names in mind, so I'm happy to merge as is.